pylon_camera/src/pylon_camera/pylon_camera_node.cpp

```diff
@@ -328,7 +328,7 @@ bool PylonCameraNode::initAndRegister()
         return false;
     }
 
-    if ( !pylon_camera_->registerCameraConfiguration() )
+    if ( !pylon_camera_->registerSoftwareTriggerConfiguration() )
```

Comment:
Should there also be an option where the registerContinuousConfiguration function is run (probably by default)?  Ideally we would keep the default settings the same so that users of the library don't have unexpected changes in what happens.

Reply:
So, it used to be `registerCameraConfiguration`, used for setting software trigger config. Because I added ability to use continuous configuration, I renamed `registerCameraConfiguration` to `registerSoftwareTriggerConfiguration`. So nothing has changed here, and default behavior is still software trigger.